voluncheer/opportunityboard/tests_views.py

```diff
@@ -49,6 +50,12 @@ def test_search(self):
         response = opportunityboard(get_request, 1)
         self.assertEqual(response.status_code, 200)
 
+    def test_search_excludes_opportunities_before_now(self):
+        """Tests that opportunities before now are not returned."""
+        with freezegun.freeze_time(self.date + datetime.timedelta(days=1)):
```

Comment:
😮

Reply:
Also, I can't figure out where self.date is coming from. Am I missing something?

Reply:
Does using `with` ensure that the freezegun is stopped after the test?

Reply:
`self.date` is coming from the `setUp` method on the base `TestCase` class.

Yes, exactly. Using `freezegun` as a context manager (with the `with` statement) boxes the functionality to the code indented below it (because Python scope is based on whitespace).

Reply:
Ah sorry I missed my own import statement :)

Thanks!